1_The_Physical/Guess_Point_Level_1_Stateless/1_Red_Green_Refactor/1_1_FizzbuzzV.2/src/fizzbuzz.ts

```diff
@@ -1,3 +1,6 @@
 export const fizzbuzz = (n: number): string => {
+    if (n < 1 || n > 100) throw numberOutofBoundsError
     return n.toString()
-}
\ No newline at end of file
+}
+
+const numberOutofBoundsError = new Error("Input must be a number between 1 and 100 inclusive")
```

Comment:
Cool! I haven't had much experience with throwing errors in TypeScript like this. That's neat.

Reply:
I definitely missed this from my Java days.

Reply:
> I definitely missed this from my Java days.

haha yeah I just finished an OOP course with Java, so it was still fresh in my mind defining custom Error classes/objects